pages/docs/tutorials/streetlights.md

```diff
@@ -52,7 +52,7 @@ Before you proceed to the next stage, you'll need to download a few things:
 In this step, we will create an AsyncAPI file to describe your API. It will help you generate the code and the documentation later on.

```

Comment:
so @BhaswatiRoy the reason you are getting that merge conflict is because this file no longer exists due to GSoD 2023 work getting merged

Reply:
Okay, thanks @alequetzalli for fixing it.